logDeployment.ps1

```diff
@@ -0,0 +1,70 @@
+param(
+    [parameter(Mandatory = $true)]
+    [string]$sha,
+    [parameter(Mandatory = $true)]
+    [string]$repoUrl,
+    [parameter(Mandatory = $true)]
+    [string]$env,
+    [parameter(Mandatory = $true)]
+    [string]$vendorKey
+)
```

Comment:
$currentTime is missing from param.

Reply:
Yeah, I have it in the script itself to log the time but I could add it here if someone wanted to overwrite it.

Reply:
O I missed that, you're good.

Reply:
I added some date stuff if you want to take a look